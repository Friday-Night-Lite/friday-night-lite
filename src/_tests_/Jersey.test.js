import React from 'react'
import {shallow} from '../enzyme'
import Jersey from '../components/Footer'
import renderer from 'react-test-renderer'
import 'jest-styled-components'


describe ('Jersey.jsx', () => {
    it('renders', () => {
        const wrapper = shallow(<Jersey/>)

        expect(wrapper.exists()).toBe(true)
    })
    it('matches snapshot', () => {
        const tree = renderer.create(<Jersey/>).toJSON()
        expect(tree).toMatchSnapshot()
    })
    it('Footer css', () => {
        const tree = renderer.create(<Jersey/>).toJSON()
        expect(tree).toMatchSnapshot()
        expect(tree).toHaveStyleRule('height', '100px')
    })
})
