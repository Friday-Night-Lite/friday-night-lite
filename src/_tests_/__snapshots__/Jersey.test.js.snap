// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jersey.jsx Footer css 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  height: 100px;
  background: #262626;
}

.c0 .container {
  width: 1200px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 h1 {
  -webkit-transition: all 1s ease;
  transition: all 1s ease;
  font-family: 'Frijole',cursive;
  color: red;
  font-size: 2rem;
}

.c0 h1:hover {
  color: white;
  cursor: pointer;
}

.c0 .cat {
  height: 50px;
}

.c1 {
  -webkit-text-decoration: none;
  text-decoration: none;
}

<div
  className="c0"
>
  <div
    className="container"
  >
    <a
      className="c1"
      href="https://www.linkedin.com/in/chazguyton/"
      target="_blank"
    >
      <h1>
        Chaz
      </h1>
    </a>
    <a
      className="c1"
      href="http://freddavison.info/"
      target="_blank"
    >
      <h1>
        Fred
      </h1>
    </a>
    <a
      className="c1"
      href="https://www.linkedin.com/in/jeramiahfields/"
      target="_blank"
    >
      <h1>
        Jeramiah
      </h1>
    </a>
    <a
      className="c1"
      href="https://www.linkedin.com/in/johnclewis8/"
      target="_blank"
    >
      <h1>
        John
      </h1>
    </a>
    <a
      className="c1"
      href="https://github.com/Friday-Night-Lite/friday-night-lite"
      target="_blank"
    >
      <img
        alt="github logo"
        className="cat"
        src="cat.png"
      />
    </a>
  </div>
</div>
`;

exports[`Jersey.jsx matches snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  height: 100px;
  background: #262626;
}

.c0 .container {
  width: 1200px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 h1 {
  -webkit-transition: all 1s ease;
  transition: all 1s ease;
  font-family: 'Frijole',cursive;
  color: red;
  font-size: 2rem;
}

.c0 h1:hover {
  color: white;
  cursor: pointer;
}

.c0 .cat {
  height: 50px;
}

.c1 {
  -webkit-text-decoration: none;
  text-decoration: none;
}

<div
  className="c0"
>
  <div
    className="container"
  >
    <a
      className="c1"
      href="https://www.linkedin.com/in/chazguyton/"
      target="_blank"
    >
      <h1>
        Chaz
      </h1>
    </a>
    <a
      className="c1"
      href="http://freddavison.info/"
      target="_blank"
    >
      <h1>
        Fred
      </h1>
    </a>
    <a
      className="c1"
      href="https://www.linkedin.com/in/jeramiahfields/"
      target="_blank"
    >
      <h1>
        Jeramiah
      </h1>
    </a>
    <a
      className="c1"
      href="https://www.linkedin.com/in/johnclewis8/"
      target="_blank"
    >
      <h1>
        John
      </h1>
    </a>
    <a
      className="c1"
      href="https://github.com/Friday-Night-Lite/friday-night-lite"
      target="_blank"
    >
      <img
        alt="github logo"
        className="cat"
        src="cat.png"
      />
    </a>
  </div>
</div>
`;
